app/models/user.rb

```diff
@@ -12,4 +12,8 @@ class User < ApplicationRecord
   def recent_posts
     posts.limit(3).order(created_at: :desc)
   end
+
+  def admin?(requested_role)
+    role == requested_role.to_s
+  end
```

Comment:
- [ ] 👏🏻 Kudos to create your `user.admin?` method inside the user model. 💯 😘 👍🏻

Reply:
Thank you! 👌